redis.tf

```diff
@@ -35,21 +35,21 @@ resource "rediscloud_subscription" "subscription" {
 
 }
 
-# resource "rediscloud_subscription_peering" "gcp-project-peering" {
-#   subscription_id = rediscloud_subscription.gcp-project.id
-#   provider_name = var.cloud_provider
-#   gcp_project_id = var.project_id
-#   gcp_network_name = var.network_name
-# }
-
-# resource "google_compute_network_peering" "gcp-network-peering" {
-#   name         = var.network_peering_name
-#   network      = var.network_name
-#   peer_network = format(
-#       "https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s",
-#       rediscloud_subscription_peering.gcp-project-peering.gcp_project_id, 
-#       rediscloud_subscription_peering.gcp-project-peering.gcp_redis_network_name 
-#     )
-# } 
+resource "rediscloud_subscription_peering" "gcp-project-peering" {
+  subscription_id = rediscloud_subscription.gcp-project.id
+  provider_name = var.cloud_provider
+  gcp_project_id = var.project_id
+  gcp_network_name = var.network_name
+}
+
+resource "google_compute_network_peering" "gcp-network-peering" {
+  name         = var.network_peering_name
+  network      = var.network_name
+  peer_network = format(
+      "https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s",
+      rediscloud_subscription_peering.gcp-project-peering.gcp_project_id, 
+      rediscloud_subscription_peering.gcp-project-peering.gcp_redis_network_name 
```

Comment:
Does this work automatically? I thought it didn't export those values based on the discussions here:

https://github.com/RedisLabs/terraform-provider-rediscloud/issues/141

Reply:
You are right, it didnt work, I uncommneted it to test create the nba-staging redis, what works is manually inputing the network and project id